.github/workflows/CI.yml

```diff
@@ -148,7 +148,10 @@ jobs:
             apt-get install -y --no-install-recommends python3 python3-pip software-properties-common
             add-apt-repository ppa:deadsnakes/ppa
             apt-get update
+            apt-get install -y python3.7-distutils
+            apt-get install -y python3.9-distutils
             apt-get install -y python3.7 python3.9
+            dpkg --install python3-distutils_3.7.0-1~18.04_all.deb
```

Comment:
Hi there.
Is it really necessary? It's supposed to be installed on line 151.

Reply:
You can also use [actions/setup-python](https://github.com/actions/setup-python) with matrix. Pretty handy.

Reply:
@manoelhc , wow this looks like a great plugin. It will definitely help. :D